src/shared/components/tc-communities/JoinCommunity/index.jsx

```diff
@@ -40,7 +41,7 @@ export default function JoinCommunity({
     <div
       className={theme.container}
     >
-      <button
+      <GenericButton
```

Comment:
Let's use default `<Button>` so that if we don't pass any particular theme via the component probs, the button will appear with the style of default buttons in that community.